docs/dev-ref/bug-bounty.md

```diff
@@ -1,16 +1,12 @@
 ---
-title: UMA Bug Bounty Program
-sidebar_label: Bug Bounty Program
+title: Audit & Bug Bounty Program
+sidebar_label: Audit & Bug Bounty Program
 ---
 
 Security of the platform is our highest priority.
 All contract code and balances are publicly verifiable, and security researchers are eligible for a bug bounty for reporting undiscovered vulnerabilities.
 
 ## Audit
-
-UMA encourages the community to audit our contracts and security; we also encourage the responsible disclosure of any issues.
-The bug bounty program is intended to recognize the value of working with the community of independent security researchers, and sets out our definition of good faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
-
 The contracts in [the common directory](https://github.com/UMAprotocol/protocol/tree/9d403ddb5f2f07194daefe7da51e0e0a6306f2c4/core/contracts/common) and [the oracle directory](https://github.com/UMAprotocol/protocol/tree/9d403ddb5f2f07194daefe7da51e0e0a6306f2c4/core/contracts/oracle) of the `/protocol` repo have been carefully audited by OpenZeppelin.
```

Comment:
This is pretty poorly formatted especially with the amount of audits that we have now. Can you bullet these? We should also add the insured bridge audits and any other ones that are missing.

Reply:
Updated, let me know if this is what you were thinking for this.

Reply:
much much better